p2p/tests/identify.rs

```diff
@@ -203,46 +202,12 @@ fn bad_node_effects(
                     multiaddr!(Https),
                 ];
 
-                let public_key = Some(SecretKey::rand().public_key());
```

Comment:
I don't know much about these tests but seems a lot if changing here, I will just assume it is correct.

Reply:
This is test to check we don't accept "bad" behavior in identify layer in this case for multiple same addresses